### DISCUSSION TITLE
Supported domain zones

### DISCUSSION THREAD

**Discussion by scriptyyy13** [deta/cloud-docs#418] [Help] [answered]
I saw in the documentation that some domain zones are not supported. I would like to know if `.su` is supported? The record itself on the domain is correct and translates to the desired IP address.So the problem is not on the side of the DNS provider. If anything I use Cloudflare and DNS only mode

**Comment by AyushSehrawat** [comment]
```
.af

.cu

.er

.gn

.ir

.kp

.lr

.nc.tr

.ru

.sd

.sl

.ss

.sy

.zw

``` 

These are the ones they mentioned...

Check here (https://docs.deta.sh/docs/micros/custom_domains/)


After setting up domain, after few minutes via some tool on internet/Linux/windows try nslookup. If it points towards the micro provided ip, then you gotta wait. Maybe micro is making certificate for it..


Also check everything in the above docs mentioned. If not message them ig

**Reply by aavshr** [reply]
hi @AyushSehrawat thanks for linking the docs and sorry but the supported domains are outdated. I will update the docs immediately.

**Comment by aavshr** [comment, marked as answer]
hi @scr1321 extremely sorry but the current supported tld docs is outdated. Unfortunately, `.su` domains are not supported by our SSL provider. We will update the docs immediately.

**Reply by scriptyyy13** [reply]
It's a pity. Are domain zones like `.ml` and `.tk` supported?

**Reply by scriptyyy13** [reply]
@aavshr can i create ssl certificate on my own and use it on deta micro? Can you do it manually, please?

**Reply by aavshr** [reply]
@scr1321 

> It's a pity. Are domain zones like .ml and .tk supported?

`.ml` and `.tk` are supported

> can i create ssl certificate on my own and use it on deta micro? Can you do it manually, please?

Sorry I can not do it manually unfortunately since our ssl provider does not support the tld. 

However, if you are using cloudflare, you can use cloudflare to generate the SSL certificate (if they support your domain) and use their [flexible](https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/flexible/) mode for the origin server.

**Reply by scriptyyy13** [reply]
okay
